website/docs/release-notes.mdx

```diff
@@ -2,6 +2,33 @@
 title: Taqueria Releases
```

Comment:
Is this redundant with the changelog that Github provides?
https://github.com/ecadlabs/taqueria/releases

My only concern is that it adds a step to the deployment process, and is another thing to maintain.